src/components/navigation/tabs/z-navigation-tab/index.tsx

```diff
@@ -10,6 +10,22 @@ import {ICONS} from "../../../icons/icons";
   styleUrl: "../navigation-tab.css",
 })
 export class ZNavigationTab {
+  /**
+   * set aria-controls attribute to tab property identifying a corresponding tabpanel
```

Comment:
Ho provato a riformulare le prima due righe di questa doc. Non credo di essere riuscito a fare meglio, ma la lascio qui lo stesso, sperando che possa aiutare: the `aria-controls` attribute refers to the id of the HTML element that has `role="tabpanel"` and that contains the actual content of this tab.